README.md

```diff
@@ -95,28 +94,6 @@ to `ts-standard` when running the command or you can add an `ignore` property to
 }
 ```
 
-## 🎛 Package.json Options
```

Comment:
I removed this section, as other `standard` engines don't document it as such and I'm not exactly sure what option to remove or add with this rewrite.